[32mINFO    [0m pytest_logger:log_util.py:49 The HTTP request was successful.